app/components/state_tag_component.rb

```diff
@@ -0,0 +1,16 @@
+class StateTagComponent < ApplicationComponent
+  attr_reader :claim
+
+  def initialize(claim:, classes: [], html_attributes: {})
+    super(classes:, html_attributes:)
+
+    @claim = claim
+  end
+end
```

Comment:
Following the [guidance from GitHub themselves](https://viewcomponent.org/viewcomponents-at-github.html#app-specific-viewcomponents), we should namespace this component if it is dependent on the model.

```ruby
Claim::StateTagComponent
```

Also, let's name it `StatusTagComponent` instead of `State` as other BAT apps tend to use `status` rather than `state`.